docker/nginx.conf

```diff
@@ -26,27 +26,27 @@ http {
 
         location / {
             proxy_pass http://127.0.0.1:4445;
+            proxy_set_header Host      $host;
+            proxy_set_header X-Real-IP $remote_addr;
+        }
+        location /dashboard {
+            # We want a relative redirect to work around reverse proxies infront of nginx
+            add_header Location "/dashboard/";
+            content_by_lua 'ngx.exit(301)';
```

Comment:
Somehow this did not work for me, I tried:

```
docker run --rm -ti --name zalenium -p 4450:4444 \
      -v /tmp/videos:/home/seluser/videos \
      -v /var/run/docker.sock:/var/run/docker.sock\
      dosel/zalenium start
```

So going to http://localhost:4450/dashboard should forward me and show the dashboard, but it did not.

Reply:
Please omit the previous comment, I was using the wrong image 🤦‍♂️

Reply:
It works!